terraform/environments/prd/aws/asg_variables.tf

```diff
@@ -0,0 +1,164 @@
+locals {
+  asgs = {
+    bpzb-tf = {
+      vpc_name = "bpzb-tf"
+      #asg_name         = "ASG"
+      sg_jh_name        = "SG_jumphost"
```

Comment:
Can we refactor this a bit. Instead of using multiple variables `sg_*` can we use something like
```
sg = {
  some_sg_1 = {
    name = "some name 1"
    description = "some description 1"
    rules = {
        ingress = [
           ......
        ]
        egress = [
           .....
        ]
    }
  }
  some_sg_2 = {
    name = "some name 2"
    description = "some description 2"
    rules = {
        ingress = [
           ......
        ]
        egress = [
           .....
        ]
    }
  }
}
```

Reply:
yes. done